modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/utils/api.ts

```diff
@@ -93,15 +103,23 @@ interface ObjectRelationship {
 	type: ObjectRelationshipType;
 }
 
-type RecipientType = 'role' | 'term' | 'user';
+interface PickListItem {
+	externalReferenceCode: string;
+	id: number;
+	key: string;
+	name: string;
+	name_i18n: LocalizedValue<string>;
+}
 
-type Recipients = {
-	bcc: string;
-	cc: string;
-	from: string;
-	fromName: LocalizedValue<string>;
-	to: LocalizedValue<string>;
-};
+interface PickList {
+	actions: Actions;
+	externalReferenceCode: string;
+	id: number;
+	key: string;
+	listTypeEntries: PickListItem[];
+	name: string;
+	name_i18n: LocalizedValue<string>;
+}
 
 const headers = new Headers({
```

Comment:
@igor-franca The `deleteFolder` function should be declared before `deleteObjectDefinitions`

Reply:
@igor-franca The function `getObjectValidationRuleById` is not sorted.